index.bs

```diff
@@ -165,37 +165,61 @@ by showing how many sites are willing to ask for intrusive and unnecessary permi
 
 <h3 id="naming-is-hard">Naming things</h3>
 
-Naming is hard! We would all like a silver-bullet for naming APIs...
-
 Names take meaning from:
 
 * signposting (the name itself)
 * use (how people come to understand the name over time)
 * context (the object on the left-hand side, for example)
```

Comment:
Should we add more descriptive suggestions about naming here - describe _what_ an API does and not _how_ etc.

Reply:
... or maybe make add that in the Future-proofing section?

Reply:
I think that's a good idea, but I'd rather not hold up this PR on it. Can you file an followup, @atanassov?